singularity/quit.sh

```diff
@@ -7,14 +7,16 @@ set -eo pipefail
 cd /sw/apps/dpdash
 npm stop
 
-# TODO: make this do a graceful exit
+# kill PM2
+./node_modules/pm2/bin/pm2 kill
```

Comment:
Sorry about the silly question--but is `node_modules/` in `/sw/apps/dpdash`?

Reply:
Yes, it is!